events-api/protobuf/v1/event.proto

```diff
@@ -34,6 +34,8 @@ enum EventAction {
   StateChange = 3;
   RebuildBegin = 4;
   RebuildEnd = 5;
+  RebuildStop = 6;
+  RebuildFail = 7;
```

Comment:
Aren't these just RebuildEnd with failure? Perhaps we can't use metadata to signify this instead of adding new events?

Reply:
RebuildEnd here is generated when rebuild is completed. Should I instead change the RebuildEnd to RebuildComplete? Or should I use single RebuildEnd (with stop, fail and complete in the metadata)?

Reply:
Yeah that's what I'm wondering.. I'm leaning more towards the latter one, end with bool flag in the event specific data. @chriswldenyer wdyt?

Reply:
Done